app/celery/tasks.py

```diff
@@ -15,6 +15,15 @@
 NOTIFY_QUEUE = 'notify-internal-tasks'
 
 
+def raise_retry_or_return_error_task_name(task, upload_filename):
```

Comment:
I was wondering about what this function name means for a bit, and then understood what it does by looking at the code.

I think adding an `and` would make it easier to understand for me. I mean like: `raise_retry_or_return_error_and_task_name`

Reply:
hmmm, that's not quite what i intended tho. its literally returning the name of the `update letters to error` task 🤔 

maybe we flip it around, will that help? we expect the retry to raise 99% of the time (as we wont often reach max retries), however, in terms of code flow it reads as getting the task name primarily.

so if we swap round it becomes eg:

`get_error_task_name_or_raise_retry`
`get_error_task_name_or_retry`

Reply:
Ahh I see now, the task is an error task, that makes sense. `get_error_task_name_or_retry` sounds good ☺️